_data/simple-icons.json

```diff
@@ -4910,6 +4910,11 @@
             "hex": "DE3723",
             "source": "https://www.namecheap.com/"
         },
+        {
+            "title": "NASA",
+            "hex": "E03C31",
```

Comment:
Thanks @adamalston, though I'm a bit confused on this colour. The SVG itself used `#DB362D`, whereas the [Wikipedia page]() suggests that `#FC3D21` is NASA Red. Taking the RGB values from [their 2006 style guide](https://users.cs.jmu.edu/bernstdh/web/common/policies/NASA_StyleGuide_Nov06.pdf) gives `#FC3D21`, so I think we should go with this.

Reply:
Per the File History comments on the [Wikipedia page](https://commons.wikimedia.org/wiki/File:NASA_Worm_logo.svg);
> Corrected color to PMC 179, used for "worm" according to this NYT article: https://www.nytimes.com/2020/04/08/science/nasa-logo-worm-spacex.html "In the manual, the color was defined as simply “NASA Red.” Today, there is a universal color matching system defined by the Pantone company, and NASA Red is Pantone color number 179."

Pantone 179 maps to `#E03C31` per the [Pantone website](https://www.pantone.com/color-finder/179-C)

If the suggestion that it is changed still stands, I can easily change it. Just didn't know what the *correct* color was for the svg.

Reply:
I'm happy to go with `#E03C31`, being as the 2006 style guide refers to the `meatball` logo instead of the `worm` wordmark. Let me review the path, and will merge if all looks good 👍🏻